rnndb/graph/g80_texture.xml

```diff
@@ -6,157 +6,312 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
-			<value value="0x35" name="Z32_S8_C8_X16_MS4_CS4" />
-			<value value="0x36" name="Z32_S8_C8_X16_MS8_CS8" />
-			<value value="0x37" name="Z24_X8_S8_C8_X16_MS4_CS12"/>
-			<value value="0x38" name="Z32_X8_C8_X16_MS4_CS12" />
-			<value value="0x39" name="Z32_S8_C8_X16_MS4_CS12" />
-			<value value="0x3a" name="Z16" />
+		<bitfield pos="30" name="PACK_COMPONENTS">
+			<doc>native 8/16bit [U|S]NORMs and FP16 components packed tightly into registers</doc>
+		</bitfield>
+		<bitfield high="29" low="27" name="W_SOURCE" type="G80_TIC_SOURCE"/>
+		<bitfield high="26" low="24" name="Z_SOURCE" type="G80_TIC_SOURCE"/>
+		<bitfield high="23" low="21" name="Y_SOURCE" type="G80_TIC_SOURCE"/>
+		<bitfield high="20" low="18" name="X_SOURCE" type="G80_TIC_SOURCE"/>
+		<bitfield high="17" low="15" name="A_DATA_TYPE" type="G80_TIC_TYPE"/>
+		<bitfield high="14" low="12" name="B_DATA_TYPE" type="G80_TIC_TYPE"/>
+		<bitfield high="11" low="9" name="G_DATA_TYPE" type="G80_TIC_TYPE"/>
+		<bitfield high="8" low="6" name="R_DATA_TYPE" type="G80_TIC_TYPE"/>
+		<bitfield high="5" low="0" name="COMPONENTS">
+			<doc>basic outline of the texel format</doc>
+			<value value="0x01" name="R32_G32_B32_A32"/>
+			<value value="0x02" name="R32_G32_B32" variants="GF100-"/>
+			<value value="0x03" name="R16_G16_B16_A16"/>
+			<value value="0x04" name="R32_G32"/>
+			<value value="0x05" name="R32_B24G8"/>
+			<value value="0x07" name="X8B8G8R8"/>
+			<value value="0x08" name="A8B8G8R8"/>
+			<value value="0x09" name="A2B10G10R10"/>
+			<value value="0x0c" name="R16_G16"/>
+			<value value="0x0d" name="G8R24"/>
+			<value value="0x0e" name="G24R8"/>
+			<value value="0x0f" name="R32"/>
+			<value value="0x12" name="A4B4G4R4"/>
+			<value value="0x13" name="A5B5G5R1"/>
+			<value value="0x14" name="A1B5G5R5"/>
+			<value value="0x15" name="B5G6R5"/>
+			<value value="0x16" name="B6G5R5"/>
+			<value value="0x18" name="G8R8"/>
+			<value value="0x1b" name="R16"/>
+			<value value="0x1c" name="Y8_VIDEO"/>
+			<value value="0x1d" name="R8"/>
+			<value value="0x1e" name="G4R4"/>
+			<value value="0x1f" name="R1"/>
+			<value value="0x20" name="E5B9G9R9_SHAREDEXP"/>
+			<value value="0x21" name="BF10GF11RF11"/>
+			<value value="0x22" name="G8B8G8R8">
+				<doc>the same as UYVY</doc>
+			</value>
+			<value value="0x23" name="B8G8R8G8">
+				<doc>the same as YUY2</doc>
+			</value>
+			<value value="0x24" name="DXT1"/>
+			<value value="0x25" name="DXT23"/>
+			<value value="0x26" name="DXT45"/>
+			<value value="0x27" name="DXN1"/>
+			<value value="0x28" name="DXN2"/>
+			<value value="0x10" name="BC6H_SF16" variants="GF100-">
+				<doc>similar to ZOH (FP16)</doc>
+			</value>
+			<value value="0x11" name="BC6H_UF16" variants="GF100-">
+				<doc>similar to ZOH (UFP16)</doc>
+			</value>
+			<value value="0x17" name="BC7U" variants="GF100-">
+				<doc>similar to ZOL (UNORM8)</doc>
+			</value>
+			<value value="0x06" name="ETC2_RGB"/>
```

Comment:
FWIW after a bit of futzing on GK208 I couldn't get this to work. Certainly an option here is that I messed it up, but I was also seeing this in dmesg:
```
[175799.299477] nouveau 0000:01:00.0: gr: TRAP ch 2 [003fbe1000 oes_compressed_[6736]]
[175799.299484] nouveau 0000:01:00.0: gr: GPC0/TPC0/TEX: 80000009
[175799.299488] nouveau 0000:01:00.0: gr: GPC0/TPC1/TEX: 80000009
```
Guessing it's GM107+?